src/components/Notifications.svelte

```diff
@@ -0,0 +1,36 @@
+<script>
+  import { Toast } from 'flowbite-svelte';
+
+  import { notificationsStore } from '../stores/notification';
+
+  const { closeNotification } = notificationsStore;
+
+  $: notifications = $notificationsStore.notifications;
+</script>
+
+<main>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#usage_notes we already have main element in our App component

Reply:
Yeah, missed that copy pasted it from `App` while creating separate component for notifications